testing/s3minio/build.gradle.kts

```diff
@@ -28,18 +28,14 @@ dependencies {
   implementation("software.amazon.awssdk:s3")
   implementation("software.amazon.awssdk:url-connection-client")
 
-  // hadoop-common brings Jackson in ancient versions, pulling in the Jackson BOM to avoid that
-  implementation(platform(libs.jackson.bom))
-  compileOnly(libs.hadoop.common)
+  implementation(libs.guava)
```

Comment:
How about renaming this module to `nessie-minio-container` to align with other similar modules?